### PR TITLE
FHAC-878:Clean-up DocQueryFanoutTestTarget Regressionsuite

### PR DESCRIPTION
**Pull request by alameluchidambaram**
1. Remove Test cases not needed.
2. UDDIConnectionInfo updated with 2 target and correlations data were set up for 2 target communities but the target was provided for one community. Since it's a Fanout test updated the Nhin Target communities in few soap scripts to have both communities.
3. Cleanup HCID description appropraite for each target community.
4. Fix assertion since 2 targets were added.

### CODE REVIEW

**Comment by alameluchidambaram** [comment]
@christophermay07 @TabassumJafri

**Comment by TabassumJafri** [comment]
Reviewed PR and good with merge

File: Product/SoapUI_Test/RegressionSuite/Standard/DocQueryFanoutTestTarget/DocQueryFanoutTestTarget-soapui-project.xml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<con:soapui-project abortOnError="false" name="DocQueryFanoutTestTarget" resourceRoot="${projectDir}" runType="SEQUENTIAL" soapui-version="5.1.2" activeEnvironment="Default" id="820ad3f4-8079-4cb8-a731-9029100b2667" xmlns:con="http://eviware.com/soapui/config">
+<con:soapui-project abortOnError="false" name="DocQueryFanoutTestTarget" resourceRoot="${projectDir}" runType="SEQUENTIAL" soapui-version="4.5.1" activeEnvironment="Default" id="820ad3f4-8079-4cb8-a731-9029100b2667" xmlns:con="http://eviware.com/soapui/config">
```
Comment (christophermay07) [inline Product/SoapUI_Test/RegressionSuite/Standard/DocQueryFanoutTestTarget/DocQueryFanoutTestTarget-soapui-project.xml:2]: We should settle on a common SoapUI version across the team, preferably in the 5.x family.

**Comment by christophermay07** [comment]
I've reviewed this PR, endorsing for merge with comments.

**Pull request merged**